CHANGELOG.md

```diff
@@ -1,10 +1,26 @@
 # Changelog
 Running changelog of releases since `3.1.1`
 
+## 9.1.0
+## Fixed
+- GetUser method now returns `User` instead of `UserGetSingleton`.  
+
+> While this is technically a breaking change, the existence and use of `UserGetSingleton` was an obvious mistake rendering the method unusable, hence the minor version increment instead of major version increment.
+
 ## 9.0.4
 ### Fixed
 - Handling of Multiple Scopes in Okta .NET SDK v9 (#753)
 
+## 9.0.3
+### Fixed
+- CreateFederatedUser creates federated user instead of user in pending activation state
+- Change Office365ApplicationSettings.App to dictionary
+
+## 9.0.1
```

Comment:
Did we miss an entry for 9.0.2?

Reply:
I'm not sure why but it looks like we skipped 9.0.2: https://github.com/okta/okta-sdk-dotnet/compare/v9.0.1...v9.0.3